skopt/acquisition.py

```diff
@@ -0,0 +1,206 @@
+
+    * `model` [sklearn estimator that implements predict with ``return_std``]:
+        The fit estimator that approximates the function through the
+        method ``predict``.
+        It should have a ``return_std`` parameter that returns the standard
+        deviation.
+
+    * `kappa`: [float, default 1.96]:
+        Controls how much of the variance in the predicted values should be
+        taken into account. If set to be very high, then we are favouring
+        exploration over exploitation and vice versa.
+        Useless if ``method`` is set to "LCB".
+
+    Returns
+    -------
+    * `values`: [array-like, shape=(X.shape[0],)]:
+        Acquisition function values computed at X.
+    """
+    # Compute posterior.
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        mu, std = model.predict(X, return_std=True)
+
+    return mu - kappa * std
+
+
+def gaussian_pi(X, model, y_opt=0, xi=0.01):
+    """
+    Use the probability of improvement to calculate the acquisition values.
+
+    The predictions form a gaussian with a certain mean and standard deviation
+    as given by the model.
+
+    The PI condition is derived by computing ``E[u(f(x))]``
+    where ``u(f(x)) = 1``, if ``f(x) > y_opt`` and ``u(f(x)) = 1``,
+    if``f(x) < y_opt``.
+
+    This means that the PI condition does not care about how "better" the
+    predictions are than the previous values, since it gives an equal reward
+    to all of them.
+
+    Parameters
+    ----------
+    * `X` [array-like, shape=(n_samples, n_features)]:
+        Values where the acquisition function should be computed.
+
+    * `model` [sklearn estimator that implements predict with ``return_std``]:
+        The fit estimator that approximates the function through the
+        method ``predict``.
+        It should have a ``return_std`` parameter that returns the standard
+        deviation.
+
+    * `y_opt` [float, default 0]:
+        Previous minimum value which we would like to improve upon.
+
+    * `xi`: [float, default=0.01]:
+        Controls how much improvement one wants over the previous best
+        values. Useful only when ``method`` is set to "EI"
+
+    Returns
+    -------
+    * `values`: [array-like, shape=(X.shape[0],)]:
+        Acquisition function values computed at X.
+    """
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        mu, std = model.predict(X, return_std=True)
+
+    values = np.zeros_like(mu)
+    mask = std > 0
+    improvement = y_opt - xi - mu[mask]
+    values[mask] = norm.cdf(improvement / std[mask])
+    return -values
+
+
+def gaussian_ei(X, model, y_opt=None, xi=0.01):
+    """
+    Use the expected improvement to calculate the acquisition values.
+
+    The predictions form a gaussian with a certain mean and standard deviation
+    as given by the model.
+
+    The PI condition is derived by computing ``E[u(f(x))]``
```

Comment:
There were two bits of docs that contained "The PI condition is derived ..." but it is fixed now.